### ISSUE TITLE
[Dashboard] Should specify the time range in job detail page for load the cluster status and scale metrics

### ISSUE THREAD

**Issue by jiwq** [ray-project/ray#41781]
### Description

In Job Detail Page display the cluster status and autoscaler metrics, and the time range is `&from=now-30m&to=now` which makes it impossible to display the cluster status of the job at that time. So I think we should replace the range to `&from={start_time}&to={end_time}`, if the job is running the `to` param is `now`.

**Comment by anyscalesam** [comment]
@scottsun94 can you triage?

**Comment by scottsun94** [comment]
Are you referring to the first graph in the screenshot? @jiwq 

![Image](https://github.com/ray-project/ray/assets/9677264/2ebb22c3-9843-4b49-aacb-6f3cb674b559)

**Comment by jiwq** [comment]
> Are you referring to the first graph in the screenshot? @jiwq
> 
> ![Image](https://private-user-images.githubusercontent.com/9677264/289702771-2ebb22c3-9843-4b49-aacb-6f3cb674b559.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzNzg0MDUsIm5iZiI6MTcwMjM3ODEwNSwicGF0aCI6Ii85Njc3MjY0LzI4OTcwMjc3MS0yZWJiMjJjMy05ODQzLTRiNDktYWFjYi02ZjNjYjY3NGI1NTkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMTJUMTA0ODI1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODU3ZGEzYjRiMTQ0OWMzMjkzNDY4ZGY5OTcwNWE2MTViMGJlYjI0ODRmNzJlNzMxZjY5YmFlZjdlZGNkODJmOSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Q7ghJomyRabrMqiH4xfPYfW6mAim7Fi_xSyyAOBkh-g)

Yes. I think all the cards in this section should be limit in the job running time range. @alanwguo Are you working on it? If not, I can submit the patch.

**Comment by scottsun94** [comment]
I'm fine with changing it for the 1st graph. The other two cards show the real-time info and we shouldn't modify them

**Comment by alanwguo** [comment]
Hi, I’m not working on this. It’d be great if you can submit a patch!
